products.d/sles_160.yaml

```diff
@@ -0,0 +1,135 @@
+id: SLES_16.0
+name: SUSE Linux Enteprise Server 16.0 Alpha
+# ------------------------------------------------------------------------------
+# WARNING: When changing the product description delete the translations located
+# at the at translations/description key below to avoid using obsolete
+# translations!!
+# ------------------------------------------------------------------------------
+description: "SUSE Linux Enterprise Server is the open, reliable, compliant, and future-proof
+  Linux Server choice that ensures the enterprise's business continuity. It is the secure and
+  adaptable OS for long-term supported, innovation-ready infrastructure running business-critical
+  workloads on-premises, in the cloud, and at the edge."
+# Do not manually change any translations! See README.md for more details.
+translations:
+  description:
+software:
+  installation_repositories:
+    - url: http://download.suse.de/ibs/SUSE:/SLFO:/Products:/SLES:/16.0/product/repo/SLES-Packages-16.0-x86_64/
+      archs: x86_64
+    - url: http://download.suse.de/ibs/SUSE:/SLFO:/Products:/SLES:/16.0/product/repo/SLES-Packages-16.0-aarch64/
+      archs: aarch64
+
+  mandatory_patterns:
+    - sles_enhanced_base
+  optional_patterns: null # no optional pattern shared
+  user_patterns: []
+  mandatory_packages:
+    - NetworkManager
+  optional_packages: null
+  base_product: SLES
+
+security:
+  lsm: null
```

Comment:
I though that default should be selinux for SLE16 or not?

Reply:
It is not available yet.

Reply:
I have changed `null` to `none`.